docs/index.zh.md

```diff
@@ -23,53 +23,71 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+<p style="margin: 30px 60px 0 60px;text-align: center" markdown="1">
+Apache Flink 是一个在无界和有界数据流上进行状态计算的框架和分布式处理引擎。 Flink 已经可以在所有常见的集群环境中运行，并以 in-memory 的速度和任意的规模进行计算。
+</p>
 
-本文档适用于 Apache Flink {{ site.version_title}} 版本。本页面最近更新于 {% build_time %}.
+<div class="row">
+<div class="col-sm-6" markdown="1">
 
-Apache Flink 是一个分布式流批一体化的开源平台。Flink 的核心是一个提供数据分发、通信以及自动容错的流计算引擎。Flink 在流计算之上构建批处理，并且原生的支持迭代计算，内存管理以及程序优化。
+### 试用 Flink
 
-## 初步印象
+如果您有兴趣使用 Flink, 可以试试我们的教程:
 
-* **代码练习**: 跟随分步指南通过 Flink API 实现简单应用或查询。
-  * [实现 DataStream 应用]({% link try-flink/datastream_api.zh.md %})
-  * [书写 Table API 查询]({% link try-flink/table_api.zh.md %})
+* [DataStream API 进行欺诈检测]({% link try-flink/datastream_api.md %})
+* [Table API 构建实时报表]({% link try-flink/table_api.md %})
+* [Python API 教程]({% link try-flink/python_table_api.md %})
+* [Flink 游乐场]({% link try-flink/flink-operations-playground.md %})
 
-* **Docker 游乐场**: 你只需花几分钟搭建 Flink 沙盒环境，就可以探索和使用 Flink 了。
-  * [运行与管理 Flink 流处理应用]({% link try-flink/flink-operations-playground.zh.md %})
+### 学习 Flink
 
-* **概念**: 学习 Flink 的基本概念能更好地理解文档。
-  * [有状态流处理](concepts/stateful-stream-processing.html)
-  * [实时流处理](concepts/timely-stream-processing.html)
-  * [Flink 架构](concepts/flink-architecture.html)
-  * [术语表](concepts/glossary.html)
+* [操作培训]({% link learn-flink/index.md %}) 包含了一系列的课程和练习，逐步介绍了，帮助你深入学习 Flink。
 
-## API 参考
+* [概念透析]({% link concepts/index.md %}) 介绍了在浏览参考文档之前你需要了解的 Flink 知识。
 
-API 参考列举并解释了 Flink API 的所有功能。
+### 获取 Flink 帮助
 
-* [DataStream API](dev/datastream_api.html)
-* [DataSet API](dev/batch/index.html)
-* [Table API &amp; SQL](dev/table/index.html)
+如果你被困住了, 可以在 [社区](https://flink.apache.org/community.html)寻求帮助。 值得一提的是，Apache Flink 的用户邮件列表一直是最活跃的 Apache 项目之一，也是一个快速获得帮助的好途径。
```

Comment:
```suggestion
如果你遇到困难了, 可以在 [社区](https://flink.apache.org/community.html)寻求帮助。值得一提的是，Apache Flink 的用户邮件列表一直 Apache 项目是最活跃的之一，也是一个快速获得帮助的好途径。
```

Reply:
I think you maybe misunderstood my comment here. others LGTM

Reply:
yes, I made some modify. @libenchao